calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
+
+  return {
+    add_number: function (number) {
+      numbers.push(number);
+    },
+    calculate: function () {
+      // Return zero if there are no numbers to calculate
+      if (numbers.length === 0) {
+        throw 'Cannot calculate a result because no numbers were given';
+      }
+
+      let answer = numbers[0];
+
+      let i = 0;
+      numbers.forEach(function (number) {
+        if (i === 0) {
+          i++;
```

Comment:
please remove this if block as it's not necessary

Reply:
Ah no. It is, but for a dumb reason. 

On line 16 "let answer = numbers[0]", which means the first number has already been handled. This block then skips the first number. 

If we didn't have this block then the first number would be operated on twice.

Recommend instead doing something like a slice to skip the first number would be neater, e.g.

let answer = numbers[0]
numbers.slice(1).forEach(...